apps/web/src/screens/Profile/components/TransactionHistory/components/TokenDetails.tsx

```diff
@@ -0,0 +1,88 @@
+import type { FC } from 'react';
+import { Image, StyleSheet } from 'react-native';
+import { Networks } from '@walless/core';
+import { appState } from '@walless/engine';
+import { Anchor, Text, View } from '@walless/gui';
+import { useSnapshot } from 'valtio';
+
+import { handleRemoveRedundantCharacters } from '../internal';
```

Comment:
name it `removeRedundantCharacters`, `handle` might be used to passing handler for on event instead